shared/chat/conversation/container.js

```diff
@@ -70,6 +73,11 @@ const mapStateToProps = (state: TypedState, {routePath, routeProps}): StateProps
     finalizeInfo = state.chat.get('finalizedState').get(selectedConversationIDKey)
     supersedes = Constants.convSupersedesInfo(selectedConversationIDKey, state.chat)
     supersededBy = Constants.convSupersededByInfo(selectedConversationIDKey, state.chat)
+    youAreReset =
+      state.chat.getIn(
+        ['inbox', selectedConversationIDKey, 'memberStatus'],
+        RPCChatTypes.commonConversationMemberStatus.active
+      ) === RPCChatTypes.commonConversationMemberStatus.reset
```

Comment:
Could you use `MessageKeyKind` you defined in types / constants to avoid importing the rpc types?